class DatasourceException implements Exception {
  final String message;

  const DatasourceException({this.message = 'Something went wrong 🙏.'});
}
